packages/orbit-components/package.json

```diff
@@ -12,6 +12,7 @@
     "cy:run": "start-server-and-test cy:dev http-get://localhost:3000 'cypress run'",
     "cy:open": "start-server-and-test cy:dev http-get://localhost:3000 'cypress open'",
     "build": "ts-node --esm config/build/index.mts",
+    "build:tailwind": "postcss ./src/tailwind.css -o ./lib/tailwind.css",
```

Comment:
why you decided to use postcss instead of default tailwindcss cli?

Reply:
As agreed, it will change to use tailwind cli but storybook will keep postcss